zrml/prediction-markets/src/migrations.rs

```diff
@@ -119,25 +120,38 @@ mod tests {
                 MarketStatus::Resolved,
             );
 
-            run_to_block(55);
+            create_test_market(
+                MarketPeriod::Timestamp(0..short_time),
+                MarketCreation::Permissionless,
+                MarketStatus::Resolved,
+            );
 
             // add one simple dispute for alreay resolved market to simulate a case
             // where there is pendig dispute(s) for resolver market and that should
             // be cleaned in storage migration.
             let market_dispute =
-                MarketDispute { at: 55, by: CHARLIE, outcome: OutcomeReport::Categorical(0) };
+                MarketDispute { at: 1, by: CHARLIE, outcome: OutcomeReport::Categorical(0) };
             let _res = crate::Disputes::<Runtime>::try_mutate(0, |disputes| {
+                let _ = disputes.try_push(market_dispute.clone());
+                disputes.try_push(market_dispute.clone())
+            });
+            let _res = crate::Disputes::<Runtime>::try_mutate(1, |disputes| {
+                let _ = disputes.try_push(market_dispute.clone());
                 disputes.try_push(market_dispute)
             });
 
             let disputes = crate::Disputes::<Runtime>::get(&0);
-            assert_eq!(disputes.len(), 1);
+            assert_eq!(disputes.len(), 2);
 
+            let disputes = crate::Disputes::<Runtime>::get(&1);
+            assert_eq!(disputes.len(), 2);
             RemoveDisputesOfResolvedMarkets::<Runtime>::on_runtime_upgrade();
 
             assert_eq!(MarketCommons::market(&0).unwrap().status, MarketStatus::Resolved);
             let disputes = crate::Disputes::<Runtime>::get(&0);
             assert_eq!(disputes.len(), 0);
+            let disputes = crate::Disputes::<Runtime>::get(&1);
+            assert_eq!(disputes.len(), 0);
```

Comment:
I don't understand the purpose of this change. What's the difference between the two markets? You just seems basically run the test twice.

If anything, it would be interesting to have two markets which resolve at different times and then see that between the resolution of the first and second market only one of them has their disputes deleted.

Reply:
The initial thought of [this](https://github.com/zeitgeistpm/zeitgeist/pull/681#discussion_r903560760) was, that the market iterator works properly. But yeah, we remove the migrations file anyways. I am okay with your suggestion.